### PR TITLE
[WHY] CoinJoin Legal Concern

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch adds a writeup by @hgergoil about the legal concerns of businesses using CoinJoin.

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -134,7 +134,8 @@ module.exports = {
           sidebarDepth: 2,
           children: [
             "/why-wasabi/WhyPrivacyImportant.md",
-            "/why-wasabi/WhyFinancialPrivacy.md"
+            "/why-wasabi/WhyFinancialPrivacy.md",
+            "/using-wasabi/LegalConcern.md"
```
Comment (yahiheb) [inline docs/.vuepress/config.js]:
```suggestion
            "/why-wasabi/LegalConcern.md"
```

**Comment by RiccardoMasutti** [comment]
Very nice! ACK

File: docs/why-wasabi/LegalConcern.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+### Competitors
+
+Everyone can agree that the pricing structure of services is a very sensitive topic and is the core of the business secret which makes a company successful.
+If all competitors can look at the transactions of the company, it also means they can adjust their own pricing structure to follow the successful market leaders in the industry, causing turbulence in competition.
+Each company has its own pricing structure which should be coming from the internal processes, know-how, and resources.
+Although external factors should be considered in pricing, they should not be the main driving force in pricing decisions.
+
+### Employees
+
+There is now a growing number of companies that are paying their employees the salaries in bitcoin due to the trustless transaction with low fees especially with international transactions.
```
Comment (PulpCattel) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
There is now a growing number of companies that are paying their employees' salaries in bitcoin due to the trustless transactions with low fees especially with international transactions. 
```

File: docs/why-wasabi/LegalConcern.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+There is now a growing number of companies that are paying their employees the salaries in bitcoin due to the trustless transaction with low fees especially with international transactions. 
+
+With transparent transaction history, employees would be aware of each other’s salaries without the company disclosing this information.
+Although there are some theories supporting a transparent salary structure, up to now the majority of the companies prefer to keep salaries as secret and obligate employees to keep salary and wage information confidential.
+Obfuscated transaction history can help companies to keep salary information confidential which can keep a relaxed atmosphere among employees with no wage pressures.
+
+### Phishing attacks
+
+Disclosing financial information for a company makes them more vulnerable to phishing attacks.
+If the exact time and amount of payment are publicly known, attackers can use this information to build trust with the company and squeeze out other crucial, business information pretending to be a trusted partner.
```
Comment (PulpCattel) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
If the exact time and amount of payment are publicly known, attackers can use this information to build trust with the company and squeeze out other crucial business information pretending to be a trusted partner.  
```

File: docs/why-wasabi/LegalConcern.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+### Phishing attacks
+
+Disclosing financial information for a company makes them more vulnerable to phishing attacks.
+If the exact time and amount of payment are publicly known, attackers can use this information to build trust with the company and squeeze out other crucial, business information pretending to be a trusted partner.  
+
+
+## Privacy perspective from the institutional investor’s point of view
+
+The Bitcoin market is considered [fairly concentrated](https://bitinfocharts.com/top-100-richest-bitcoin-addresses.html) in the sense that our of the millions of Bitcoin addresses, less than 1% holds 87% of bitcoins.
```
Comment (PulpCattel) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
The Bitcoin market is considered [fairly concentrated](https://bitinfocharts.com/top-100-richest-bitcoin-addresses.html) in the sense that out of the millions of Bitcoin addresses, less than 1% holds 87% of bitcoins.
```

File: docs/why-wasabi/LegalConcern.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+In case of a law enforcement request, all data can be provided, since the deposit history of the user in question is still available.
+The only thing CoinJoin does, in this case, is protecting the user from third-party observers, while the service provider is still enabled to provide data for law enforcement if requested.
+
+### FinCEN guidance on legal framework
+
+The Financial Crimes Enforcement Network (FinCEN) is a bureau of the United States Department of the Treasury that collects and analyzes information about financial transactions.
+It is claimed to be in order to combat domestic and international money laundering, terrorist financing, and other financial crimes.
+Its latest regulation regarding cryptocurrencies is in effect since May 2019.
+The regulation places money transmitting in the center of the discussion.
+Money transmitters must develop a certain process in terms of reporting and monitoring to help to combat money laundering and terrorist financing.
```
Comment (yahiheb) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
Money transmitters must develop a certain process in terms of reporting and monitoring to help combat money laundering and terrorist financing.
```

File: docs/why-wasabi/LegalConcern.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+FinCEN guidance explicitly states that those persons and businesses who provide the delivery, communication, or network access services used by a money transmitter to support money transmission services are NOT considered money transmitters.
+Furthermore, in section 4.5 anonymity-enhanced cryptocurrency transactions are detailed and concluded that they are subject to the same AML and KYC regulations as any other transactions.
```
Comment (yahiheb) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
Furthermore, in section 4.5 anonymity-enhanced cryptocurrency transactions are detailed and concluded that they are subject to the same AML and KYC regulations as any other transactions.
```

File: docs/why-wasabi/LegalConcern.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+This is the time when a provider could preserve its users’ privacy by applying CoinJoin on these funds.
+In case of a withdrawal, coinjoined UTXOs could be sent to the user.
+Again, this user is already KYC-ed, the provider has full knowledge about the user.
+In case of a law enforcement request, all data can be provided, since the deposit history of the user in question is still available.
+The only thing CoinJoin does, in this case, is protecting the user from third-party observers, while the service provider is still enabled to provide data for law enforcement if requested.
+
+### FinCEN guidance on legal framework
+
+The Financial Crimes Enforcement Network (FinCEN) is a bureau of the United States Department of the Treasury that collects and analyzes information about financial transactions.
+It is claimed to be in order to combat domestic and international money laundering, terrorist financing, and other financial crimes.
```
Comment (yahiheb) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
It is claimed that its job is to combat domestic and international money laundering, terrorist financing, and other financial crimes.
```

File: docs/why-wasabi/LegalConcern.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+This is obviously not intentional and even though the personal information might be protected, the financial information is nevertheless leaked due to the pseudonym nature of the system.
+This information has a similar sensitivity as name, address, gender, etc. so it should be handled with great care.
+
+### CoinJoin as a privacy-preserving tool
+
+Since the Bitcoin blockchain is transparent, transactions cannot be kept secret.
+Currently, one prevalent solution in Bitcoin to gain privacy is to CoinJoin the different coins to obfuscate their history.
+By the use of this tool, a custodial service provider can at least start to protect its users’ financial privacy.
+
+## Privacy perspective from the companies point of view
```
Comment (yahiheb) [inline docs/why-wasabi/LegalConcern.md]:
```suggestion
## Privacy perspective from the companies' point of view
```

File: docs/why-wasabi/LegalConcern.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+
+The essence of competitive advantages to keep and hold business secrets.
+Each and every successful company has developed its own mechanism to deliver the best product and services to its customers.
+To reach this, they usually use a secret sauce, called the business secret to deliver the results.
+If everything, even financial payment becomes fully transparent this business secret is evaporating, leaving a lot of confusion, anger and jealousy behind.
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/why-wasabi/LegalConcern.md]:
```suggestion
If anything, when a financial payment becomes fully transparent this business secret is evaporating
```

File: docs/why-wasabi/LegalConcern.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+The Bitcoin market is considered [fairly concentrated](https://bitinfocharts.com/top-100-richest-bitcoin-addresses.html) in the sense that our of the millions of Bitcoin addresses, less than 1% holds 87% of bitcoins.
+
+![](/AddressDistribution.png)
+
+Institutional investors and cryptocurrency exchanges are holding large sums of bitcoin on a blockchain where transactions are completely transparent.
+These so-called whales sometimes need to move large amounts of coins for any reason which can often lead to unintentional price manipulation.
+If a large bitcoin holding is on the move, it can always raise concern in the media regarding the future of bitcoin, causing unnecessary price fluctuations.
+In the case of unintentional price manipulation, it is difficult to prove there has been no insider trading on the institutional side, which can cause loss of faith from the clients.
+
+Using obfuscated transaction history in large bitcoin transactions can help to keep the market stable and avoid unintentional price manipulations.
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/why-wasabi/LegalConcern.md]: well, it kinda depends here...

If "larger" means more vByte, then yes, *B*itcoin.
If "larger" means more sats, then no, *b*itcoin.
As the latter is ment, I think that *b*itcoin is proper.

**Pull request merged**